### DISCUSSION TITLE
React Native windows release build is not working.

### DISCUSSION THREAD

**Discussion by AhmedMustafa505** [microsoft/react-native-windows#11956] [Q&A] [answered]
Hello, I'm implementing a new desktop application and I'm newly using react native windows, when I build the app in a debug mode it works fine, but when I tried to build a release build and create AppPackages to install on other devices it doesn't work at all, it shows a black screen. even when I ran the release build on my local machine. 
any solutions ?

**Comment by chrisglein** [comment, marked as answer]
Closing this discussion as that there's an issue here: #11957
We'll focus analysis there.
